tests/unit/tls_test.cc

```diff
@@ -45,7 +45,9 @@
 #include "loopback_socket.hh"
 #include "tmpdir.hh"
 
+#ifndef SEASTAR_WITH_TLS_OSSL
 #include <gnutls/gnutls.h>
+#endif
```

Comment:
haven't looked at the rest of the file, but is the header needed at all?

Reply:
I believe this https://github.com/michael-redpanda/seastar/blob/1548f346ab0d350bbd9eef173083622eb7d0febd/tests/unit/tls_test.cc#L775 comes from that header.